src/xma/src/xmaapi/CMakeLists.txt

```diff
@@ -38,7 +38,6 @@ target_link_libraries(xma2api
   stdc++
   xml2
   boost_filesystem
-  boost_iostreams
```

Comment:
I'm look at this requirement. If I can get rid of this.

Reply:
Please let me know.  I am baffled why this dependency was added, there is nothing in the code that depends on it.

Reply:
Ok please merge this change. I made small change and this is no longer required.